rel-1.0.0/windowsservercore/onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet-nightly:windowsservercore
```

Comment:
I think all of the references to images from the dotnet-nightly repo should be using the fully qualified tag names (e.g. rel-1.0.0-windowsservercore) so that they don't break as the shortened tags are moved to other images over time.